posts/business-leaders-and-digital-transformation.md

```diff
@@ -0,0 +1,59 @@
+
+## Product-management practices
+1. In order to communicate business value behind software development initiatives, effective product management practices must be deployed. This includes communication to business executives throughout the product development lifecycle, a process that consists of several steps from numerous teams:
+    * Storyboards, customer feedback, market analysis, success metrics or benchmarks (such as KPIs or OKRs), and team performance updates.
+    * Leadership needs to have a firm understanding of what works and what doesn't. By creating that transparency, executives can support the management styles that are most effective at lower levels of the organization, allow the teams to pivot and iterate quickly, leverage team member's skill sets, enable the team to become thought leaders and ultimately enrich management at every level.
+
+## Product-development practices
+1. Metrics around release schedules, testing and the extensibility of application interfaces allow business leaders to get insights into the entire lifecycle of a software development initiative. These include:
+    * Schedules allow them to have a baseline to measure progress against.
+    * Testing provides insight into how the development is progressing.
+    * Application interfaces allow them to think ahead in terms of strategy and growth. 
+
+## Product-delivery practices
+1. In cloud-based development, the use of tools to gather data-driven insights, polling customer engagement through the use of selective feature releases, and leveraging open-source tools to reduce development time and the cost of maintenance. 
+    * "Show me the data" - Data driven business insights have become a prerequisite when justifying decisions at the executive level. Integrating business analytics tools is a critical investment so a business can react quickly to changing market trends. Agility and responsiveness can differentiate a successful business. 
+    * A/B testing - When mapping paths for development, leadership often encounters uncertainty - a fork in the road if you will - where existing data is not sufficient to determine the best path forward. Running experiments with real users allows leaders to gauge customer engagement. This method is critical to reducing ambiguity and in making data driven-decisions. 
+    * Community-driven development - the utilization of open-source software. "Why reinvent the wheel?" There are no extra points in business for building out a solution that already exists. By leveraging open-source software, businesses can benefit on multiple fronts. The first being that the software is tried and tested, with robust documentation. The second being that costs for development can be significantly reduced by leveraging crowd-sourcing for maintenance and feature development. By focusing on differentiating features, rather than the development and maintenance of a custom codebase, companies can move faster and do more, for less.
```

Comment:
```suggestion
    * Community-driven development: The utilization of open-source software. _"Why reinvent the wheel?"_ There are no extra points in business for building out a solution that already exists. By leveraging open-source software, businesses can benefit on multiple fronts. The first being that the software is tried and tested with robust documentation. The second being that costs for development can be significantly reduced by leveraging crowd-sourcing for maintenance and feature development. By focusing on differentiating features, rather than the development and maintenance of a custom codebase, companies can move faster and do more for less.
```